toolium/behave/env_utils.py

```diff
@@ -204,11 +203,19 @@ def __execute_steps_by_action(self, context, action):
                     context.execute_steps(u'''%s%s''' % (GIVEN_PREFIX, self.__remove_prefix(item)))
                     self.logger.debug(u'step defined in pre-actions: %s' % repr(item))
                 except Exception as exc:
-                    if action in [ACTIONS_BEFORE_FEATURE]:
-                        self.feature_error = True
-                    elif action in [ACTIONS_BEFORE_SCENARIO]:
-                        self.scenario_error = True
+                    self.error = True
                     self.logger.error(exc)
```

Comment:
use  `self.logger.warn(exc)` instead of  `self.logger.error(exc)` to avoid  throw the raise exception

Reply:
it is not a warn but an error